src/session/sessionReducer.js

```diff
@@ -37,6 +37,11 @@ export type SessionState = {|
   isActive: boolean,
   isHydrated: boolean,
   lastNarrow: ?Narrow,
+
+  /**
+   * Whether the /register request is in progress; i.e., on startup, or on
+   * re-init following a dead event queue after 10 minutes of inactivity.
+   */
   loading: boolean,
   needsInitialFetch: boolean,
```

Comment:
nit: add blank line after `loading`

Otherwise it looks as if the comment might apply to all the next few properties.